test/OpenTelemetry.Tests/Trace/ExportProcessorTest.cs

```diff
@@ -14,14 +14,11 @@
 // limitations under the License.
 // </copyright>
 
-using System.Collections.Generic;
 using System.Diagnostics;
-using OpenTelemetry.Exporter;
-using OpenTelemetry.Exporter.Console;
-using OpenTelemetry.Trace;
+using OpenTelemetry.Tests;
 using Xunit;
 
-namespace OpenTelemetry.Tests.Trace
+namespace OpenTelemetry.Trace.Tests
```

Comment:
Thanks for fixing this!

Reply:
Yeah not sure how it got missed in the first place.